web/modeltr/enums.py

```diff
@@ -35,8 +35,23 @@ class RequestState(StrEnumBase):
     TIMEOUTED = 'timeouted'
     ABORTED = 'aborted'
 
-    def has_finished(self):
+    def has_finished(self) -> bool:
```

Comment:
is it going to be working in python 3.6?

Reply:
Yes, I'm testing locally with 3.6 ;)

Reply:
👍